sys-init/DuplicateCaps_SI.thy

```diff
@@ -256,7 +256,7 @@ lemma distinct_card':
 (* FIXME, move higher *)
 lemma distinct_length_filter':
   "distinct xs \<Longrightarrow> length [x\<leftarrow>xs. P x] = card {x \<in> set xs. P x}"
-  by (metis distinct_length_filter set_conj_Int_simp inf_commute)
+  by (metis distinct_card' distinct_filter set_filter)
```

Comment:
unexpected change for this PR

Reply:
ah, but it should be expected :-). The lemma `set_conj_Int_simp` for some reason was proved in `WPC.thy` and is unused in l4v apart from this one file. So this change is a consequence of cleaning up `WPC.thy`.